sbnana/SBNAna/ananue/PAC2020/SBND/helper_pur_slc_cumulative.h

```diff
@@ -29,7 +28,7 @@ const Binning kPdgBinning = Binning::Simple(5000, -2500, 2500);
 // // In this example, we are making the following Spectra
 std::vector<PlotDef> plots = {
   { "count", "", Binning::Simple(3, 0, 3), kCounting },
-  { "bestenergy", "Best plane energy (MeV)", kEnergyBinning, kRecoShower_BestEnergy },
+  { "bestenergy", "Best plane energy (MeV)", kEnergyBinningGeV, kRecoShower_BestEnergy },
```

Comment:
```suggestion
  { "bestenergy", "Best plane energy (GeV)", kEnergyBinningGeV, kRecoShower_BestEnergy },
```
If I get it right, these variables are now in GeV and they are going to be shown as such.

Reply:
Agreed. I clicked the tempting "commit suggestion" button since this is so trivial.